doc/src/modules/physics/mechanics/examples/multi_degree_freedom_holonomic_system.rst

```diff
@@ -0,0 +1,93 @@
+=========================================
+Multi Degree of Freedom Holonomic System
+=========================================
+
+In this example we demonstrate the use of the functionality provided in
+:mod:`sympy.physics.mechanics` for deriving the equations of motion (EOM) of a holonomic
+system that includes both particles and rigid bodies with contributing forces and torques,
+some of which are specified forces and torques. The system is shown below:
+
+.. image:: multidof-holonomic.*
+   :align: center
+
+The system will be modeled using ``JointsMethod``. First we need to create the
+``dynamicsymbols`` needed to describe the system as shown in the above diagram.
+In this case, the generalized coordinates `q1` represent lateral distance of block from wall,
+`q2` represents ngle of the compound pendulum from vertical, `q3` represents angle of the simple
```

Comment:
These variables should be proper latex, e.g. ```:math:`q_2` ```.

Reply:
Done!

![Screenshot (52)](https://user-images.githubusercontent.com/45732670/127316397-35d77c77-eef5-4069-be3f-dd6b4abcabb9.png)